<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Bijans-MacBook-Pro.local" ignored="0" name="Default test" tests="3" failures="0" timestamp="10 Mar 2017 15:41:11 GMT" time="8.164" errors="0">
  <testcase name="@BeforeTest setUp" time="0.0" classname="testCases.Validate_LoginAndLogout">
    <failure type="java.lang.NoSuchMethodError" message="org.openqa.selenium.os.CommandLine.find(Ljava/lang/String;)Ljava/lang/String;">
      <![CDATA[java.lang.NoSuchMethodError: org.openqa.selenium.os.CommandLine.find(Ljava/lang/String;)Ljava/lang/String;
	at org.openqa.selenium.phantomjs.PhantomJSDriverService.findPhantomJS(PhantomJSDriverService.java:232)
	at org.openqa.selenium.phantomjs.PhantomJSDriverService.createDefaultService(PhantomJSDriverService.java:181)
	at org.openqa.selenium.phantomjs.PhantomJSDriver.<init>(PhantomJSDriver.java:105)
	at org.openqa.selenium.phantomjs.PhantomJSDriver.<init>(PhantomJSDriver.java:95)
	at testCases.Validate_LoginAndLogout.setUp(Validate_LoginAndLogout.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.beforeRun(TestRunner.java:634)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase name="@AfterTest endSession" time="0.0" classname="testCases.Validate_LoginAndLogout">
    <skipped/>
  </testcase> <!-- @AfterTest endSession -->
  <testcase name="A_Launch" time="0.003" classname="testCases.Validate_LoginAndLogout">
    <skipped/>
  </testcase> <!-- A_Launch -->
  <testcase name="B_Login" time="0.002" classname="testCases.Validate_LoginAndLogout">
    <skipped/>
  </testcase> <!-- B_Login -->
  <testcase name="C_Logout" time="0.001" classname="testCases.Validate_LoginAndLogout">
    <skipped/>
  </testcase> <!-- C_Logout -->
</testsuite> <!-- Default test -->
